src/node/interfaces.cpp

```diff
@@ -58,6 +58,7 @@
 #include <any>
 #include <memory>
 #include <optional>
+#include <string>
```

Comment:
Nit: I believe this addition might be unnecessary, considering that other instances of `std::string` in the file seem to work fine without it